djangocms_frontend/contrib/modal/templates/djangocms_frontend/bootstrap5/modal-inner.html

```diff
@@ -0,0 +1,11 @@
+{% load cms_tags frontend %}
+<{{ instance.tag_type }}{{ instance.get_attributes }}>
+{% with parent=instance %}
+    {% for plugin in instance.child_plugin_instances %}
+        {% with forloop as parentloop %}{% render_plugin plugin %}{% endwith %}
+    {% endfor %}
+{% endwith %}
+{% if instance.inner_type == "modal-header" %}
```

Comment:
Should a close button not be optional?

Reply:
Hi @fsbraun, you are right. We should add an option to show  an x in the header, a close button  in the footer, both or none. Is this what you expect?